client/app/states/dashboard/dashboard.state.js

```diff
@@ -188,6 +188,31 @@ function StateController($state, RequestsState, ServicesState, definedServiceIds
     $state.go('services');
   };
 
+  vm.navigateToRetiredServicesList = function() {
```

Comment:
Rather than three functions that complete the same action, why not one function that acts in three different ways?

food for thought